scripts/download-blocks.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+set -ex
+
+BLOCKS_DIR="./blocks"
+touch "$BLOCKS_DIR"/block_urls
+
+MIN_LENGTH=1
+MAX_LENGTH=200
+
+OUT_DIR=.
+
+for LENGTH in `seq $MIN_LENGTH $MAX_LENGTH`
+do
+    echo "gs://mina_network_block_data/mainnet-${LENGTH}-*.json" >> "$BLOCKS_DIR"/block_urls
+done 2>/dev/null
+
+gsutil -m cp -n -I "$BLOCKS_DIR" < "$BLOCKS_DIR"/block_urls
```

Comment:
I think we can/should get rid of `gsutil` and download from our own bucket

See https://github.com/Granola-Team/mina-indexer/issues/211#issuecomment-2010888237

Reply:
That would be cool. Would these scripts be targeted to Granolas or to the broader community as well?

Reply:
I think targeting the broader community would be a good idea.

Reply:
That's a different issue and shouldn't block this PR